required/latex-lab/latex-lab-footnotes.dtx

```diff
@@ -1981,14 +1974,19 @@
   {
     \tag_mc_end_push:
 %    \end{macrocode}
-% test if a footnote is allowed, if not move up to the document structure.
+% test if a footnote is allowed, if not move up to the next sect or
+% the document structure.
 %    \begin{macrocode}
     \tag_check_child:nnTF {FENote}{pdf2}
       { 
         \tag_struct_begin:n { tag=footnote }
       }
       {  
-        \tag_struct_begin:n { tag=footnote,parent=\l_@@_dflt_struct_tl }
+        \tag_struct_begin:n 
+          { 
+            tag=footnote,
+            parent=\int_max:nn{2}{\tag_get:n{current_Sect}+0} 
```

Comment:
In case the `\tag_get:n{current_Sect}` returns nothing (if the latex-lab-sec code is loaded this should never happen, but it could happen if someone only loads the footnote code and currently I still try to avoid to many dependencies between the latex-lab-modules.)

Reply:
In that case you should document that in the code I would suggest